coremltools/models/neural_network/builder.py

```diff
@@ -1445,55 +1469,63 @@ def add_inner_product(
     ):
         """
         Add an inner product layer to the model.
-        Refer to the **InnerProductLayerParams** message in specification (NeuralNetwork.proto) for more details.
+        Refer to the ``InnerProductLayerParams`` message in the specification 
+        (NeuralNetwork.proto) for more details.
 
         Parameters
         ----------
         name: str
             The name of this layer.
         W: numpy.array or bytes()
-            Weight matrix of shape (output_channels, input_channels)
-            If W is of type bytes(), i.e. quantized, other quantization related arguments must be provided as well (see below).
+            Weight matrix of shape ``(output_channels, input_channels)``.
+            If ``W`` is of type ``bytes()`` (quantized), other quantization 
+            related arguments must be provided as well (see below).
         b: numpy.array
-            Bias vector of shape (output_channels, ).
+            Bias vector of ``shape (output_channels, )``.
         input_channels: int
             Number of input channels.
         output_channels: int
             Number of output channels.
         has_bias: boolean
-            Whether the bias vector of this layer is ignored in the spec.
+            Whether the bias vector of this layer is ignored in the ``spec``.
```

Comment:
"spec" here means the protobuf that get generated. I don't think that should have double backticks.

Reply:
The full word would be "specification" correct? I was thrown off by the use of "spec" as a parameter in `__init__(input_features=None, output_features=None, mode=None, spec=None, nn_spec=None, disable_rank5_shape_mapping=False, training_features=None, use_float_arraytype=False)` at the very top of [NeuralNetworkBuilder](https://apple.github.io/coremltools/source/coremltools.models.neural_network.html#coremltools.models.neural_network.builder.NeuralNetworkBuilder). In the latter case, I use `spec` -- is that correct? Thanks!